Core.DomainServices/Repositories/Contract/ItContractRepository.cs

```diff
@@ -19,35 +19,6 @@ public class ItContractRepository : IItContractRepository
             _systemUsageRepository = systemUsageRepository ?? throw new ArgumentNullException(nameof(systemUsageRepository));
         }
 
-        public IQueryable<ItContract> GetBySystemUsageAssociation(int systemUsageId)
-        {
-            var usage = _systemUsageRepository.GetByKey(systemUsageId);
-            if (usage == null)
-            {
-                throw new ArgumentException("No system usage found", nameof(systemUsageId));
-            }
-
-            var idsOfContractsWithDirectAssociations = usage.Contracts.Select(x => x.ItContractId);
-            var interfaceExhibitUsages = usage.ItInterfaceExhibitUsages.ToList();
-            var interfaceUsages = usage.ItInterfaceUsages.ToList();
```

Comment:
Enig i at disse to skal væk. Men resten er da stadig gældende? Systemusages kan godt sættes sammen med kontrakter